packages/plugins/plugin-deck/src/DeckPlugin.tsx

```diff
@@ -283,23 +287,13 @@ export const DeckPlugin = ({
       window.addEventListener('popstate', handleNavigation);
```

Comment:
should plugins all extend @dxos/context/Resource to make unregistering simpler.
e.g.,

```ts
this._ctx.onDispose(Event.addEventListener(window, 'popstate', () => {});
```